docs/source/user_guide/gcm_based_inference/answering_causal_questions/quantify_intrinsic_causal_influence.rst

```diff
@@ -6,51 +6,64 @@ By quantifying intrinsic causal influence, we answer the question:
     How strong is the causal influence of a source node to a target node
     that is not inherited from the parents of the source node?
 
-Naturally, descendants will have a zero intrinsic influence on the target node.
+Naturally, descendants will have a zero intrinsic influence on the target node. This method is based on the paper:
+
+    Dominik Janzing, Patrick Blöbaum, Lenon Minorics, Philipp Faller, Atalanti Mastakouri. `Quantifying intrinsic causal contributions via structure preserving interventions <https://arxiv.org/abs/2007.00714>`_
+    arXiv:2007.00714, 2021
+
+Let's consider an example from the paper to understand the type of influence being measured here. Imagine a schedule of
+three trains, ``Train A, Train B`` and ``Train C``, where the departure time of ``Train C`` depends on the arrival time of ``Train B``,
+and the departure time of ``Train B`` depends on the arrival time of ``Train A``. Suppose ``Train A`` typically experiences much
+longer delays than ``Train B`` and ``Train C``. The question we want to answer is: How strong is the influence of each train
+on the delay of ``Train C``?
+
+While there are various definitions of influence in the literature, we are interested in the *intrinsic causal influence*,
+which measures the influence of a node that has not been inherited from its parents, that is, the influence of the noise
+of a node. The reason for this is that, while ``Train C`` has to wait for ``Train B``, ``Train B`` mostly inherits the delay from
+``Train A``. Thus, ``Train A`` should be identified as the node that contributes the most to the delay of ``Train C``.
+
+See the :ref:`Understanding the method <understand-method>` section for another example and more details.
 
 How to use it
 ^^^^^^^^^^^^^^
 
-To see how the method works, let us generate some data.
+To see how the method works, let us generate some data following the example above:
 
 >>> import numpy as np, pandas as pd, networkx as nx
 >>> from dowhy import gcm
->>> from dowhy.gcm.uncertainty import estimate_variance
->>> np.random.seed(10)  # to reproduce these results
 
->>> X = np.random.normal(loc=0, scale=1, size=1000)
->>> Y = 2*X + np.random.normal(loc=0, scale=1, size=1000)
->>> Z = 3*Y + np.random.normal(loc=0, scale=1, size=1000)
+>>> X = abs(np.random.normal(loc=0, scale=5, size=1000))
```

Comment:
Do X, Y, Z represent **delays**? If so, then delays can also be negative, the train may arrive faster than usual. A realistic distribution would then be a Gaussian distribution which has a non-zero mean with negative observations ~3 standard deviations away.

Reply:
In that case, we would also need to introduce something like ``Y = max(0, X) + N``, since the following train will not leave earlier if the train is faster.

I am not sure if we really need to make it that complicated here in the "how to use the method" section, since this is mostly to show how the API works? I only tried to link it a bit to the previous example to not have it completely abstract.